GiniVision/Classes/Core/GiniConfiguration.swift

```diff
@@ -423,6 +423,13 @@ import UIKit
         NSLocalizedStringPreferred("ginivision.onboarding.fourthPage",
                                    comment: "Text on the fourth page of the onboarding screen")
     
+    /**
+     Sets the text on the fifth onboarding page.
+     */
+    public var onboardingFifthPageText =
```

Comment:
It's a bit strange to have five method for modifying each element in an array. It'd opt for having an array property where all values are set at the same time.

I know it's already part of the API, but we can consider deprecating these methods and adding the one described above.

P.S. Same applies for the suggestions.